cli/remotebazel/remotebazel.go

```diff
@@ -561,13 +559,14 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 		req.Timeout = timeout.String()
 	}
 
+	log.Printf("\nWaiting for available remote runner...\n")
```

Comment:
Doesn't have to be in this PR but we might want to wait until we've waited 2-3s without receiving the invocation ID before we print this message. Feels like this will be pretty noisy in the case where the invocation starts immediately (ideally we want it to feel ~instant and we wouldn't print this message ever)

Reply:
sounds good :+1: